index.test-d.ts

```diff
@@ -133,3 +133,32 @@ convertElement()
 convertElement(null)
 convertElement(undefined)
 expectError(convertElement<Article>())
+
+declare const node: unknown
+
+/* Type assertion */
+if (isElement(node)) {
+  expectType<Element>(node)
+}
+
+if (
+  isElement<Section>(
+    node,
+    (node): node is Section => 'tagName' in node && node.tagName === 'section'
```

Comment:
In this assertion, the `node` parameter will always be an element. This function isn’t called otherwise.
In the function body, `'tagName' in node` can then be omitted, as it’s always true